tests/test_main.py

```diff
@@ -184,8 +184,8 @@ def test_infer_next_step(tmp_dir, mocker):
 def test_custom_steps(tmp_dir):
     dvclive.init("logs")
 
-    steps = [0, 62, 1000]
-    metrics = [0.9, 0.8, 0.7]
+    steps = [10, 0, 62, 1000]
+    metrics = [1.0, 0.9, 0.8, 0.7]
```

Comment:
```suggestion
    steps = [0, 62, 1000]
    metrics = [0.9, 0.8, 0.7]
```

Maybe we could add a separate test with a more realistic scenario, for example:

```python
def test_log_reset_with_step_0(tmp_dir):
    for i in range(3):
        dvclive.log("train_m", 1, step=i)

    for i in range(3):
        dvclive.log("val_m", 1, step=i)

    assert read_history("dvclive", "train_m") == ([0, 1, 2], [1, 1, 1])
    assert read_history("dvclive", "val_m") == ([0, 1, 2], [1, 1, 1])
```

Reply:
You are right.
Looks much better. 🙂

Reply:
I added  separate test according to your suggestion. (Your example 1:1)